raiden/messages.py

```diff
@@ -59,7 +60,10 @@ def assert_transfer_values(identifier, token, recipient):
 
 
 def decode(data):
-    klass = CMDID_TO_CLASS[data[0]]
+    try:
+        klass = CMDID_TO_CLASS[data[0]]
+    except KeyError:
+        raise InvalidProtocolMessage('Invalid message type (data[0] = {})'.format(data[0]))
```

Comment:
please, add the full data encoded as hexadecimal, otherwise the logs won't have all the necessary data to help debugging.

Reply:
It's printed out when the exception is caught, full log looks like this:

`WARNING:raiden.network.transport	Can't decode: Invalid message type (data[0] = 0x41) (data=b'ABCDEF\n', len=7)`